### PR TITLE
Home page UI completed (2)

### PR DESCRIPTION
**Pull request by Amit-Aditaya**
pushing my work to the new new changes that you made

### CODE REVIEW

File: lib/features/tabs/ui/screens/components/category_items.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class CategoryItems extends StatelessWidget {
+  const CategoryItems({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/category_items.dart]: This Container should be a Generic component that behaves like a CARD with rounded corners and shape.

File: lib/features/tabs/ui/sections/category_section.dart [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      //  height: 275,
+      margin: const EdgeInsets.only(bottom: 52),
+      width: double.infinity,
+      color: Colors.transparent,
+      child: Column(children: [
+        const CategorySectionHeader(),
+        const SizedBox(height: 20),
```
Comment (rifat0153) [inline lib/features/tabs/ui/sections/category_section.dart]: Use constant variables to add size info. e.g. SizedBox(height: kPadding * 2.5) or single constant value to reuse everywhere.

File: lib/features/tabs/ui/screens/components/recently_ordered_items.dart [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                                  ),
+                                  fontSize: 12,
+                                  fontWeight: FontWeight.w500),
+                            ),
+                          ),
+                        ]),
+                  ))
+            ],
+          ),
+          Positioned(
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/recently_ordered_items.dart]: Avoid using hard-coded values for positioning and sizing.

File: lib/features/tabs/ui/screens/components/recently_ordered_items.dart [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class RecentlyOrderedItems extends StatelessWidget {
+  const RecentlyOrderedItems({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Container(
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/recently_ordered_items.dart]: Should be another type of Generic Card.

File: lib/features/tabs/ui/screens/components/top_picks_item.dart [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class TopPicksItem extends StatelessWidget {
+  const TopPicksItem({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/top_picks_item.dart]: Make a Generic Component in the shared/components/card folder. There should be three different cards. One for categories, one for top picks, and one for recent orders.

File: lib/features/tabs/ui/screens/components/top_picks_section_header.dart [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+class TopPicksSectionHeader extends StatelessWidget {
+  const TopPicksSectionHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: const [
+        Text(
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/top_picks_section_header.dart:14]: Use A Shared Text Widget like AppText where you get to pass style or have predefined styles.

File: lib/features/tabs/ui/screens/components/top_picks_section_header.dart [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+          'Top Picks',
+          style: TextStyle(
+              fontSize: 28,
```
Comment (rifat0153) [inline lib/features/tabs/ui/screens/components/top_picks_section_header.dart:17]: Use Material 3 sizing from Theme object like -> Theme.of(context).textTheme.bodyLarge. Look at Figma file to know which component uses what material text.

File: lib/shared/components/appbars/main_appbar.dart [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import 'package:elvan/shared/components/constants/app_colors.dart';
```
Comment (rifat0153) [inline lib/shared/components/appbars/main_appbar.dart:1]: A more appropriate name is preferred than main_appbar.

**Review by rifat0153 — CHANGES_REQUESTED** [review]
Look at the requested changes and solve them. If you have any questions dm me on FB.

**Comment by Amit-Aditaya** [comment]
changes done except texts, need to discuss about it

**Pull request closed without merging**